gtsfm/configs/deep_front_end.yaml

```diff
@@ -46,12 +46,14 @@ SceneOptimizer:
 
     data_association_module:
       _target_: gtsfm.data_association.data_assoc.DataAssociation
-      reproj_error_thresh: 100
-      min_track_len: 2
-      mode:
-        _target_: gtsfm.data_association.data_assoc.TriangulationParam
-        value: 0 # 0 corresponds to NO_RANSAC
-      num_ransac_hypotheses: 20
+      triangulation_options:
+        _target_: gtsfm.data_association.point3d_initializer.TriangulationOptions
+        reproj_error_threshold: 10
+        min_track_len: 2
+        mode:
+          _target_: gtsfm.data_association.point3d_initializer.TriangulationParam
+          value: 1 # 0 corresponds to NO_RANSAC
```

Comment:
same here -- would be nice to explain what 1 corresponds to. If we are using RANSAC in this config, what do we need the new `deep_front_end_astronet` config for?

Reply:
AstroNet performs better with a higher RANSAC threshold (100 pixels as opposed to 10) due to the quality of the averaged poses.